src/providers/tenderly-simulation-provider.ts

```diff
@@ -396,11 +399,15 @@ export class TenderlySimulator extends Simulator {
         this.tenderlyProject
       );
 
+      metric.putMetric('TenderlySimulationSwapRouter02Requests', 1, MetricLoggerUnit.Count);
+
       const before = Date.now()
 
-      const resp = (
+      const { data: resp, status: httpStatus } = (
         await axios.post<TenderlyResponseSwapRouter02>(url, body, opts)
-      ).data;
+      );
+
+      metric.putMetric('TenderlySimulationSwapRouter02ResponseStatus', httpStatus);
```

Comment:
I think we need to separate these, add the status in the metric name rather than the value